Automattic/wc-calypso-bridge
# Add task header for "Launch your store" task

### Changes proposed in this Pull Request:



Close #1014

Add task header for "Launch your store" task

![Screenshot 2023-03-02 at 12 53 33](https://user-images.githubusercontent.com/4344253/222334513-aaae35fd-cfd6-470d-9af5-a58ec4382c45.png)


figma:
9bBYcxxQPSbof8E1CFLHmx-fi-714%3A51335







### How to test the changes in this Pull Request:




1. Change [this line](https://github.com/Automattic/wc-calypso-bridge/blob/e98dd37e51c7213b2524cd350de4857ad4fc0d9b/includes/tasks/class-wc-calypso-task-add-domain.php#L90) to return true
2. Change [this line](https://github.com/Automattic/wc-calypso-bridge/blob/e98dd37e51c7213b2524cd350de4857ad4fc0d9b/includes/tasks/class-wc-calypso-task-launch-site.php#L98) to return true
3. Set `launch-status` option to false if needed
4. Go to `WooCommerce > Home`
5. Complete all tasks except launch your store task
6. Confirm "Launch your store" task has a task header, and it matches the design
7. Click on "Launch your store" button
8. You should see the "Ready to launch your store?" page



### Other information:




### FOR PR REVIEWER ONLY:
